deps/src/main/java/com/microsoft/azure/sdk/iot/deps/serializer/DeviceParser.java

```diff
@@ -480,84 +465,99 @@ public void setLastActivityTime(Date lastActivityTime)
      */
     public void setCloudToDeviceMessageCount(long cloudToDeviceMessageCount)
     {
-        //Codes_SRS_DEVICE_PARSER_34_029: [This method shall set the value of this object's cloudToDeviceMessageCount equal to the provided value.]
         this.cloudToDeviceMessageCount = cloudToDeviceMessageCount;
     }
 
     /**
-     * Getter for ManagedBy
+     * Getter for ManagedBy.
      *
-     * @return The value of ManagedBy
+     * @return The value of ManagedBy.
      */
     public String getManagedBy()
     {
-        //Codes_SRS_DEVICE_PARSER_34_018: [This method shall return the value of this object's Status.]
         return managedBy;
     }
 
     /**
-     * Setter for ManagedBy
-     * @param managedBy the value to set managedBy to
+     * Setter for ManagedBy.
+     *
+     * @param managedBy The value to set ManagedBy to.
      */
     public void setManagedBy(String managedBy)
     {
-        //Codes_SRS_DEVICE_PARSER_34_017: [This method shall set the value of this object's Status equal to the provided value.]
         this.managedBy = managedBy;
     }
 
     /**
-     * Getter for capabilities
+     * Getter for Capabilities.
      *
-     * @return The value of capabilities
+     * @return The value of Capabilities.
      */
     public DeviceCapabilitiesParser getCapabilities()
     {
-        //Codes_SRS_DEVICE_PARSER_34_018: [This method shall return the value of this object's IotEdge.]
         return capabilities;
     }
 
     /**
-     * Setter for capabilities
-     * @param capabilities the value to set capabilities to
+     * Setter for Capabilities.
+     *
+     * @param capabilities The value to set Capabilities to.
      */
     public void setCapabilities(DeviceCapabilitiesParser capabilities)
     {
-        //Codes_SRS_DEVICE_PARSER_34_017: [This method shall set the value of this object's capabilities equal to the provided value.]
         this.capabilities = capabilities;
     }
 
     /**
-     * Get the security scope for this device
-     * @return the security scope for this device
+     * Get the device scope for this device.
+     *
+     * @return The device scope for this device.
      */
     public String getScope()
     {
         return this.scope;
     }
 
     /**
-     * Set the security scope for this device
-     * @param scope the security scope to set
+     * Set the device scope for this device.
+     *
+     * @param scope The device scope to set.
      */
     public void setScope(String scope)
     {
         this.scope = scope;
     }
+
+    /**
+     * Get the parent scopes for this device.
+     *
+     * @return The parent scopes for this device.
+     */
+    public List<String> getParentScopes()
+    {
+        return this.parentScopes;
+    }
+
+    /**
+     * Sets the parent scope for this device.
+     *
+     * @param parentScopes The parent scopes to set.
+     */
+    public void setParentScopes(List<String> parentScopes) { this.parentScopes = parentScopes; }
```

Comment:
nit: should this go into a new line, to follow the same format as other setters

Reply:
The other setters in this file follow this style. Are you asking me to update this file to match the others?

Reply:
I was referencing `setScope` in L544 that has a different format (braces and setter in different lines). It is possible that this might not be consistent throughout the file.

Reply:
I'm not seeing inconsistency in this file, except for lines that would otherwise be too long have been broken out long form.

Reply:
Maybe I couldn't communicate what I meant:

https://github.com/Azure/azure-iot-sdk-java/blob/ae7ae6b7515afb0927e2208858c2e01f83b6ae09/deps/src/main/java/com/microsoft/azure/sdk/iot/deps/serializer/DeviceParser.java#L544-L547

vs 

https://github.com/Azure/azure-iot-sdk-java/blob/ae7ae6b7515afb0927e2208858c2e01f83b6ae09/deps/src/main/java/com/microsoft/azure/sdk/iot/deps/serializer/DeviceParser.java#L564

Reply:
You want the former style? In my file, `setScope` method is not formatted the way you've pasted it.

Reply:
I found what I'm seeing is an IDE feature that makes it look like it is one line. Tim helped me see that, and I've resolved it.